components/post_view/post_context.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+const PostContext = React.createContext({
+
+    // Post component event handler that should be
+    // called when any child component opens/closes a
+    // menu type component.
+    handleMenuOpened: null,
```

Comment:
What about making this a bit more generic to be applicable for a wider range of popup's? It could then also be used for things like the emoji picker, which isn't a menu but is a popup. Just a thought for consideration. 🙂

```suggestion
    handlePopupOpened: null,
```

Reply:
Make sense. Done 👍